src/amber/controller/filters.cr

```diff
@@ -70,8 +80,9 @@ module Amber::Controller
       filters[filter.precedence] << filter
     end
 
-    def run(precedence : Symbol, action : Symbol)
+    def run(precedence : Symbol, action : Symbol, except_action : Symbol | Nil = nil)
       filters[precedence].each do |filter|
+        next if filter.action == :except && filter.try(&.excepts).try(&.includes?(except_action))
```

Comment:
this is a lot on one line, can this block be refactored to avoid using .try and be more readable?

Reply:
Got it. Made it more readable and took the the calls to try away in latest.